### PR TITLE
Wrote section for Ready to Drive Sound

### PR DESCRIPTION
**Pull request by segerpeter07**

### CODE REVIEW

File: esf_main.tex [hunk 1]
```diff
@@ -5,9 +5,63 @@
...
+and the current to 20 mA.
+\subsubsection{Wiring, cables, current calculations, connectors}
+%Describe wiring, show schematics, describe connectors and cables and show useful data regarding the wiring.
+When the shutdown circuit closes and activates the AIRs, the car is in ready to drive mode. As soon as the
+car is in this mode, the CAN system will activate the ready to drive sound node to send a positive output
+that powers the relay for 2 seconds, thus letting the buzzer sound for 2 seconds. The resistor will be current-limiting and act in the place of a fuse.
+
+
+\begin{figure}[h]
+	\includegraphics[width=\linewidth]{Schematic}
```
Comment (aehoppe) [inline esf_main.tex]: Please use a more descriptive file name. This file name will create conflicts down the line.

File: esf_main.tex [hunk 2]
```diff
@@ -5,9 +5,63 @@
...
+	\begin{equation}
+		\centering
+		P=0.021A*250V
+	\end{equation}
+	\begin{equation}
+		\centering
+		P=5.25W
+	\end{equation}
+%End Equations
+According to these equations, a 1.2K$\Omega$ resistor will function as a current limiting resistor.
```
Comment (aehoppe) [inline esf_main.tex]: It'll have to be like a 10W power resistor as well. You should talk about where you're going to mount it to ensure that it gets adequate cooling.

File: esf_main.tex [hunk 3]
```diff
@@ -5,9 +5,63 @@
...
+\subsubsection{Position in car}
+%Provide CAD-renderings showing all relevant parts. Mark the parts in the rendering, if necessary.
+The ready to drive sound will be located in the dashboard enclosure shown in Figure !!!!. The buzzer will be mounted
+to the exterior and underside of this enclosure. It must be contained outside of the box so that the buzzer is
+loud enough and underneath to assist with water proofing of the container
```
Comment (aehoppe) [inline esf_main.tex]: Is the buzzer waterproof?

**Review by aehoppe — CHANGES_REQUESTED** [review]
Please see my comments for a few changes I'd recommend. However, I think we'll have to redesign this entirely. There will not be TS wiring going into the dashboard. Let's set up a meeting to talk about this.

File: esf_main.tex (hunk 3 above)
Comment (segerpeter07) [reply esf_main.tex]: The new one I chose is 100% waterproof

File: esf_main.tex [hunk 4]
```diff
@@ -13,55 +13,28 @@
...
 %Describe your concept of the RTDS, how is the sound produced, what are the parameters for activating the RTDS, etc.
-The Ready to Drive sound is located as a component of the dashboard subsystem, and contains a buzzer (\href{http://cirlw3yrzg4afc1x47lv2k18.wpengine.netdna-cdn.com/wp-content/uploads/2016/12/WT-1205-new.pdf}{Link: Soberton Inc. WT-1205}). The buzzer automatically makes a noise when given power, with the loudness proportional
-to the voltage. The last step in the start up sequence will notify the CAN system it is time for the ready
-to drive sound. Then the corresponding node on the buzzer will close a relay between TS+, after a 1.2 K$\Omega$
-resistor, and the buzzer for two seconds. The resistor limits the voltage over the buzzer to 48V
-and the current to 20 mA.
+The Ready to Drive sound is located as a component of the dashboard subsystem, and contains a buzzer (\href{http://www.puiaudio.com/pdf/XL-5530L-LW300-S-R.pdf}{Link: PUI Audio Inc. XL-5530L-LW300-S-R}). The buzzer automatically makes a noise when given power, with the loudness proportional
+to the voltage. The last step in the start up sequence will notify the CAN system that it is time for the ready
```
Comment (aehoppe) [inline esf_main.tex]: This should say that the BMS will notify the Dashboard CAN node when the shutdown circuit is complete.

File: esf_main.tex [hunk 5]
```diff
@@ -13,55 +13,28 @@
...
-		\centering
-		P=5.25W
-	\end{equation}
-%End Equations
-According to these equations, a 1.2K$\Omega$ resistor will function as a current limiting resistor.
+\begin{figure}[H]
+	\includegraphics[width=\linewidth]{FormulasForDistanceAndSoundLevel}
+	\caption{Inverse Square Law equations needed to calculate the volume at a certain distance}
+\end{figure}
+Using the inverse square law for sound pressure levels, 123db at 10cm will translate to 117db at 2m which is well over the minimum SPL.
```
Comment (aehoppe) [inline esf_main.tex]: I'm not so thrilled with the volume of this buzzer, it's well into the territory where hearing it at close range will be dangerous to hearing. Can you find one a little quieter, like 100 dB?

File: esf_main.tex [hunk 6]
```diff
@@ -13,55 +13,28 @@
...
+	\caption{Inverse Square Law equations needed to calculate the volume at a certain distance}
+\end{figure}
+Using the inverse square law for sound pressure levels, 123db at 10cm will translate to 117db at 2m which is well over the minimum SPL. 
 \subsubsection{Position in car}
 %Provide CAD-renderings showing all relevant parts. Mark the parts in the rendering, if necessary.
 The ready to drive sound will be located in the dashboard enclosure shown in Figure !!!!. The buzzer will be mounted
-to the exterior and underside of this enclosure. It must be contained outside of the box so that the buzzer is
-loud enough and underneath to assist with water proofing of the container
+to the exterior and front side of this enclosure. It must be contained outside of the box so that the buzzer is
+loud enough and is waterproof, so it doesn't require any extra protection besides a waterproof O-ring around the mounting point.
```
Comment (aehoppe) [inline esf_main.tex]: We'll probably end up using some waterproof silicone sealant like RTV for this for a more permanent seal once we're confident

**Review by aehoppe — CHANGES_REQUESTED** [review]
Your buzzer should be on the high side of the N-FET so that the FET changes the buzzer's connection to ground, not power.

File: esf_main.tex [hunk 7]
```diff
@@ -13,14 +13,13 @@
 \subsection{Ready-To-Drive-Sound (RTDS)}
 \subsubsection{Description}
 %Describe your concept of the RTDS, how is the sound produced, what are the parameters for activating the RTDS, etc.
-The Ready to Drive sound is located as a component of the dashboard subsystem, and contains a buzzer (\href{http://www.puiaudio.com/pdf/XL-5530L-LW300-S-R.pdf}{Link: PUI Audio Inc. XL-5530L-LW300-S-R}). The buzzer automatically makes a noise when given power, with the loudness proportional
-to the voltage. The last step in the start up sequence will notify the CAN system that it is time for the ready
-to drive sound. This will then provide voltage to the gate of an N-FET which will provide 12V to the buzzer.
+The Ready to Drive sound is located as a component of the dashboard subsystem, and contains a buzzer (\href{http://www.mallory-sonalert.com/specifications/STA20502.PDF}{Link: Mallory Sonalert Products Inc. STA20502}). The buzzer automatically makes a noise when given power, with the loudness proportional
```
Comment (aehoppe) [inline esf_main.tex]: Please provide a reference with \ref to a subsection of the appendix that shows a piece of the data sheet with volume charcteristics, and also has a hyperlink to the full datasheet, as per last year's ESF.

File: esf_main.tex [hunk 8]
```diff
@@ -31,10 +30,10 @@ \subsubsection{Wiring, cables, current calculations, connectors}
 	\includegraphics[width=\linewidth]{FormulasForDistanceAndSoundLevel}
 	\caption{Inverse Square Law equations needed to calculate the volume at a certain distance}
 \end{figure}
-Using the inverse square law for sound pressure levels, 123db at 10cm will translate to 117db at 2m which is well over the minimum SPL. 
+Using the inverse square law for sound pressure levels, 97db at 122cm will translate to 93db at 2m which is a loud enough volume without inducing harm to the listener.
```
Comment (aehoppe) [inline esf_main.tex]: Also link to the appendix

**Review by aehoppe — CHANGES_REQUESTED** [review]

File: esf_main.tex [hunk 9]
```diff
@@ -30,10 +30,20 @@ \subsubsection{Wiring, cables, current calculations, connectors}
...
+Using the inverse square law as referenced in the appendix  for sound pressure levels \ref{Ready to Drive}, 97db at 122cm will translate to 93db at 2m which is a loud enough volume without inducing harm to the listener. 
 \subsubsection{Position in car}
 %Provide CAD-renderings showing all relevant parts. Mark the parts in the rendering, if necessary.
 The ready to drive sound will be located in the dashboard enclosure shown in Figure !!!!. The buzzer will be mounted
 to the exterior and front side of this enclosure. It must be contained outside of the box so that the buzzer is
 loud enough. The speaker is waterproof, so it doesn't require any extra protection besides using an RTV silicone seal for final instillation. 
+
+
+\newpage
+\subsection{Ready to Drive (APPENDIX SECTION)}\label{Ready to Drive}
```
Comment (aehoppe) [inline esf_main.tex]: Can you put this as a subsection after Section 11 (Appendix)? Sorry to keep running this back and forth.

**Review by aehoppe — CHANGES_REQUESTED** [review]
Just one more thing, please put the appendix section under Section 11 with the same subsection name as the section of the ESF as per last year's document.

**Pull request closed without merging**